snippet_uiautomator/uiautomator.py

```diff
@@ -116,7 +116,9 @@ def __init__(
   @property
   def _is_apk_installed(self) -> bool:
     """Checks if the snippet apk is already installed."""
-    all_packages = self._device.adb.shell(['pm', 'list', 'package'])
+    all_packages = self._device.adb.shell(
+        ['pm', 'list', 'packages', self._configs.snippet.package_name]
```

Comment:
Thanks for the change. With this change, we can now directly check if the return value is empty or not.

```
  @property
  def _is_apk_installed(self) -> bool:
    return bool(ad.adb.shell(['pm', 'list', 'packages', self._configs.snippet.package_name]))
```

Reply:
Very rarely, but keeping the following grep is helpful for full package name matching, just in case there returns multiple packages with similar names.